felix/bpf-gpl/parsing.h

```diff
@@ -87,6 +92,10 @@ static CALI_BPF_INLINE int parse_packet_ip(struct cali_tc_ctx *ctx) {
 
 	return PARSING_OK;
 
+ipv6_packet:
+	// IPv6 has a fixed header format, so we don't need to check the header length here
```

Comment:
Sadly, not quite true.  The IP header is fixed but hey have a new concept: extension headers, which can be inserted between the IP header and the payload header.  We'll need to deal with that, I'm think there are several in common use.

Reply:
I was thinking to handle IP options (if needed) in IPv6 programs, but considering that we can exploit the same thing for Ipv4 options, I will handle it in the same parsing file.